src/settings/mofa_settings.rs

```diff
@@ -11,210 +13,347 @@ live_design! {
     use crate::shared::widgets::*;
 
     ICON_EDIT = dep("crate://self/resources/icons/edit.svg")
+    ICON_DELETE = dep("crate://self/resources/icons/delete.svg")
+    ICON_REMOTE = dep("crate://self/resources/images/globe_icon.png")
+    ICON_LOCAL = dep("crate://self/resources/images/laptop_icon.png")
+
+    ICON_SUCCESS = dep("crate://self/resources/images/circle_check_icon.png")
+    ICON_LOADER = dep("crate://self/resources/images/loader_icon.png")
+    ICON_FAILURE = dep("crate://self/resources/images/refresh_error_icon.png")
+
+    // Not making this based on <Icon> because button does not support images
+    // (and these SVGs are too complex for Makepad's SVG support)
+    ConnectionStatusIcon = <View> {
+        visible: false
+        cursor: Hand
+        width: Fit, height: Fit
+        icon = <Image> {
+            width: 18, height: 18
+            // Override the color of the icon
+            draw_bg: {
+                instance tint_color: #B42318
+
+                fn get_color_scale_pan(self, scale: vec2, pan: vec2) -> vec4 {
+                    let tex_color = sample2d(self.image, self.pos * scale + pan).xyzw;
+                    // Use the alpha channel from the texture but replace RGB with our tint color
+                    // Assuming the icon is black/white with transparency
+                    return vec4(
+                        self.tint_color.rgb * tex_color.a,
+                        tex_color.a
+                    );
+                }
+            }
+        }
+    }
 
-    pub MofaSettings = {{MofaSettings}} {
-        width: Fill, height: Fit
+    MofaServerItem = {{MofaServerItem}} {
+        width: Fill, height: 60
+        show_bg: true
+        draw_bg: {
+            color: #f
+        }
         flow: Down
-        spacing: 20
 
-        <Label> {
-            draw_text:{
-                text_style: <BOLD_FONT>{font_size: 16}
-                color: #000
+        separator = <View> {
+            margin: {left: 20, right: 20, top: 0, bottom: 10}
+            width: Fill,
```

Comment:
stupid comment crossing my mind: views are Fill, Fill by default. We are not being explicit about height being Fill, but we are explicit on width being Fill. Shall we remove `width: Fill`? add height?

Reply:
I guess we can come up with a convention about that (and other DSL things like comma usage, etc.
In this case the height is 1 so both are explicit, I'll remove the width for now.